4-tools/manual.ipynb

```diff
@@ -501,7 +501,7 @@
    "source": [
     "## Slideshows\n",
     "\n",
-    "Our presentations use a sort of \"slideshow\" version of Jupyter notebooks. There are a few ways this can be done, and we have used [RISE](https://rise.readthedocs.io/en/stable/#).\n",
+    "Our presentations use a sort of \"slideshow\" version of Jupyter notebooks. There are a few ways this can be done, and we have used [RISE](https://rise.readthedocs.io/en/master/).\n",
```

Comment:
If you don't care which version of the documentation the link points to then not specifying the version at all should be even more robust:
```suggestion
    "Our presentations use a sort of \"slideshow\" version of Jupyter notebooks. There are a few ways this can be done, and we have used [RISE](https://rise.readthedocs.io/).\n",
```